### ISSUE TITLE
Support for AlloyDB?

### ISSUE THREAD

**Issue by ff-sdesai** [yesdeepakverma/pysql-beam#12]
Hi,
It's not a bug but had a question and did not find any link for raising a question. Does this library also support AlloyDB which is an extension of Postgre-sql? I could not find it in the documentation so, wanted to confirm

**Comment by yesdeepakverma** [comment]
We use python postgresdb writer to write to Postgres. We have not tested it
on Alloydb.

**Comment by ff-sdesai** [comment]
All right, thanks. Will try it out with AlloyDB

**Comment by ff-sdesai** [comment]
I am trying to use the latest version in my pyproject.toml. However, I am getting following error-
`Because pysql-beam (0.2) depends on psycopg2-binary (2.8.2). and no versions of pysql-beam match >0.2,<0.3, pysql-beam (>=0.2,<0.3) requires 				psycopg2-binary (2.8.2). So, because up-trend-analyzer-service depends on both psycopg2-binary (^2.9.9) and pysql-beam (~0.2), version solving failed.`
psycopg2-binary is more than 4 years old. Is there any reason why pysql-beam is using 4 years old version of psycopg2-binary?

**Comment by yesdeepakverma** [comment]
Yes, I was worried about similar errors. It was developed few years ago due
to my current work policies note able to make it current. I will try to
test it in coming days to work with latest version . Contributions are very
much needed for this tool to grow.
